controllers/getUsers.controller.js

```diff
@@ -0,0 +1,11 @@
+const { selectUsers } = require("../models/getUsers.model");
+
+exports.getUsers = (req, res) => {
+  selectUsers()
+    .then((result) => {
+      res.status(200).send({ users: result });
```

Comment:
Why not just call what's returned from your model users? Then you wouldn't have to have a different key!